ChaosMod/Main.h

```diff
@@ -3,12 +3,14 @@
 #include "DebugMenu.h"
 #include "EffectDispatcher.h"
 #include "TwitchVoting.h"
+#include "Effects/MetaEffectInfo.h"
 
 #include <Windows.h>
 #include <memory>
 #include <map>
 
 inline std::map<EffectType, EffectData> g_enabledEffects;
+inline MetaEffectInfo g_metaInfo = MetaEffectInfo();

```

Comment:
No need to set it specifically, it'll initialize as a `MetaEffectInfo` object and run the default constructor anyways.

Reply:
Thanks, that's a pet peeve of mine, especially when "style guidelines" specifically recommend the extra line version. :D